actuator/src/main/java/org/tron/core/vm/OpCode.java

```diff
@@ -577,6 +586,19 @@
 
   ISCONTRACT(0xd4, 1, 1, OpCode.Tier.ExtTier),
 
+  STAKE(0xd5, 2, 1, OpCode.Tier.ExtTier),
+
+  UNSTAKE(0xd6, 0, 1, OpCode.Tier.ExtTier),
+
+  WITHDRAWREWARD(0xd7, 1, 1, OpCode.Tier.ExtTier),
+
+  REWARDBALANCE(0xd8, 1, 1, OpCode.Tier.ExtTier),
+
+  ISSRCANDIDATE(0xd9, 1, 1, OpCode.Tier.ExtTier),
```

Comment:
better call it `ISWITNESS`.
The code base has no `sr candidate` but `witness candidate`.

Reply:
`ISWITNESS` will be used in future plan. We define it as a different meaning compare to this scenario.

Reply:
It already introduces inconsistency.